helix-term/src/commands/lsp.rs

```diff
@@ -1367,13 +1368,25 @@ fn compute_inlay_hints_for_view(
                         None => continue,
                     };
 
-                let label = match hint.label {
-                    lsp::InlayHintLabel::String(s) => s,
-                    lsp::InlayHintLabel::LabelParts(parts) => parts
-                        .into_iter()
-                        .map(|p| p.value)
-                        .collect::<Vec<_>>()
-                        .join(""),
+                let label = {
```

Comment:
Stylewise we avoid `let` blocks because they lead to increased indentation. It's fine to let `label` be `mut` for this `for` loop scope